veggie-selector/options.html

```diff
@@ -18,6 +18,7 @@
         <option value="eggplants" class="option"> &#127814; eggplants</option>
         <option value="potatoes" class="option">&#129364; potatoes</option>
         <option value="tomatoes" class="option">&#127813; tomatoes</option>
+        <option value="pumpkins" class="option">&#127875; pumpkins</option>
```

Comment:
Same as the other comments, please move above tomatoes ⬆️  🍅

Reply:
Moved all the pumpkins above the tomatoes! 🎃